### DISCUSSION TITLE
Is Surface Kerenel 5.1.15 compatible with Ubuntu 20?

### DISCUSSION THREAD

**Discussion by bentur1** [linux-surface/linux-surface#519] [Q&A]
Hi,

Well, I am running Ubuntu 18 on surfacebook2 with the Surface kernel 5.1.15

I was wondering if it compatible with the lastest Ubuntu 20?

Will I have to upgrade the kernel?

Thx Github community

**Comment by qzed** [comment]
That kernel is quite old and outdated. I'd recommend upgrading to a recent one (e.g. 5.13.7 should be available in a couple of hours) and keeping that up-to-date via our repos.

As to whether that (5.1.15) will work with Ubuntu 20, I can't say. As far as I can tell, Ubuntu 20 ships with a 5.10 kernel. It's probably likely that most things will work, but some things might break/require a newer kernel if they've explicitly been compiled for e.g. 5.10.

**Reply by bentur1** [reply]
Thank you QZED!

Wishing you a luminous day
